botocore/docs/service.py

```diff
@@ -56,7 +56,13 @@ def table_of_contents(self, section):
         section.style.table_of_contents(title='Table of Contents', depth=2)
 
     def client_api(self, section):
-        ClientDocumenter(self._client).document_client(section)
+        examples = None
+        try:
+            examples = self._session.get_examples(self._service_name)
```

Comment:
Can we just move `get_examples` into this class rather than on the `session`?  I'd really like to only add methods to the `session` when necessary and when it's something a customer could conceivably call.  I don't really see a customer needing to call `session.get_examples`.